repo/packages/D/dynatrace/1/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_48x48px.png",
+    "icon-medium": "https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_96x96px.png",
+    "icon-large": "https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_256x256px.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "agent": "dynatrace/oneagent:latest"
```

Comment:
please use a tagged version of an image

Reply:
Would `dynatrace/oneagent:mesos` work for you?

Reply:
done